coord/src/dist_aggregate.c

```diff
@@ -309,6 +309,13 @@ static void buildMRCommand(RedisModuleString **argv, int argc, int profileArgs,
     }
   }
 
+  // check for timeout argument and append it to the command
+  int timeout_index = RMUtil_ArgIndex("TIMEOUT", argv, argc);
```

Comment:
Should start looking for TIMEOUT from `argv + 3 + profileArgs`? (e.g., if index name is TIMEOUT)